README.md

```diff
@@ -46,6 +46,15 @@ that a device's first boot will include the proper configuration:
   # echo <TOKEN> > ./data/fio-api-token
 ~~~
 
+If you are using a Foundries domain which is not `app.foundries.io`, then
```

Comment:
Let's emphasize that means meds. e.g:

If you are using a Foundries domain which is not app.foundries.io, (i.e. MEDs), then